sdk/go/common/workspace/project.go

```diff
@@ -965,3 +966,32 @@ func save(path string, value interface{}, mkDirAll bool) error {
 	//nolint:gosec
 	return os.WriteFile(path, b, 0o644)
 }
+
+// To mitigate an import cycle, we define this here.
+const PulumiTagsConfigKey = "pulumi:tags"
```

Comment:
Do we need this defined in core.go as well? It feels more _project_ then _core_ and isn't actually being used in core.go unlike here.

Reply:
Begrudgingly putting up with this for now 🥲